pkg/guestagent/kubernetesservice/kubernetesservice.go

```diff
@@ -55,15 +55,14 @@ func (s *ServiceWatcher) getServiceInformer() cache.SharedIndexInformer {
 
 func (s *ServiceWatcher) Start() {
 	retryInterval := 10 * time.Second
-	//nolint:staticcheck // SA1019: wait.PollInfinite is deprecated
-	wait.PollInfinite(retryInterval, func() (done bool, err error) {
+	pollImmidiately := false
```

Comment:
```suggestion
	const pollImmidiately = false
```

Reply:
Updated in [7c9d353](https://github.com/lima-vm/lima/pull/1748/commits/7c9d353f52247d325631bbb47ae1a55c5cd2e6c3)